README.md

```diff
@@ -58,8 +61,7 @@ Before you begin, ensure you have the following:
 
 - **AWS CLI**: Installed and configured with appropriate permissions.
 - **Docker**: Installed on your local machine.
-- **IAM Role**: The function retrieves credentials from AWS Secrets Manager to make authorized requests to the ingest service, ensuring that the upload process is secure and properly authenticated.
-  Ensure you have a role with necessary permissions for Lambda execution, S3 and Secrets Manager access.
+- **IAM Role**: Ensure you have a role with necessary permissions for Lambda execution, S3 access, and SNS.
```

Comment:
I would either write the explicit permissions required, or provide a json sample.